scripts/ci/pack_soljson.sh

```diff
@@ -18,9 +18,9 @@ echo "Packing $soljson_js and $soljson_wasm to $output."
     echo -n 'var Module = Module || {}; Module["wasmBinary"] = '
     echo -n '(function(source, uncompressedSize) {'
     # Note that base64DecToArr assumes no trailing equals signs.
-    cat "${script_dir}/base64DecToArr.js"
+    cpp "${script_dir}/base64DecToArr.js" | grep -v "^#.*"
     # Note that mini-lz4.js assumes no file header and no frame crc checksums.
-    cat "${script_dir}/mini-lz4.js"
+    cpp "${script_dir}/mini-lz4.js" | grep -v "^#.*"
```

Comment:
Since @cameel doesn't want these to really be minified, I guess this is the easiest way to get rid of the comments :-).

Reply:
I'd actually feel better about minifying them compared to selectively removing the comments :-D.